scripts/run-django-dev.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# This script runs the django app, waiting on the react applications to build
+
+python3 manage.py collectstatic --noinput --clear
```

Comment:
I thought `collectstatic` needed to run after the `wait-for`, once the build directories have completed?

How will collectstatic / webpack build work in production? I saw this note in the django-webpack-loader docs (https://github.com/django-webpack/django-webpack-loader#usage-in-production):

>⚠️ Heroku is one platform that automatically runs collectstatic for you, so you need to set DISABLE_COLLECTSTATIC=1 environment var. Instead, you must manually run collectstatic after running webpack. In Heroku, this is achieved with a post_compile hook. You can see an example on how to implement this flow on [django-react-boilerplate](https://github.com/vintasoftware/django-react-boilerplate/tree/master/bin).